custom_components/homewhiz/translations/select.de.json

```diff
@@ -40,7 +49,11 @@
     },
     "homewhiz__WASHER_SPIN": {
       "SPIN_NO_SPIN": "Nicht schleudern",
-      "SPIN_RINSE_HOLD": "Sp\u00fclstopp"
+      "SPIN_RINSE_HOLD": "Sp\u00fclstopp",
+      "SPIN_600": 600,
+      "SPIN_800": 800,
+      "SPIN_1200": 1200,
+      "SPIN_1400": 1400
```

Comment:
I tried this locally: The error is gone, and all select files can now be used for selection. However, I'm seeing this for the spin entity:
![image](https://user-images.githubusercontent.com/15891478/209721275-421a2988-d0d4-4b94-9cac-4d00a6bb0533.png)
I believe the numbers here need to be strings.